playbooks/tkurki/cassiopeia.yml

```diff
@@ -46,6 +46,7 @@
         - "-A FORWARD -i eth1 -o eth0 -m state --state RELATED,ESTABLISHED -j ACCEPT"
         - "-A FORWARD -i eth0 -o eth1 -j ACCEPT"
       nat_router_inet_interface: eth1
+    # Update: it is recommended to NOT use signalk anymore, use signalk-npm or signalk-docker instead.
```

Comment:
I'll remove the signalk role separately, so let's leave this comment out.

Reply:
I can't figure out how i remove this commit from this PR. I saw that you already removed the file. So i think we can close this? (just learning how to work with PRs).